src/cli-validator/utils/printResults.js

```diff
@@ -91,8 +101,19 @@ module.exports = function print(
       chalk.cyan(`  Total number of errors   : ${stats.errors.total}`)
     );
     console.log(
-      chalk.cyan(`  Total number of warnings : ${stats.warnings.total}\n`)
+      chalk.cyan(`  Total number of warnings : ${stats.warnings.total}`)
     );
+    if (stats.infos.total > 0) {
+      console.log(
+        chalk.cyan(`  Total number of infos    : ${stats.infos.total}`)
+      );
+    }
+    if (stats.hints.total > 0) {
+      console.log(
+        chalk.cyan(`  Total number of hints    : ${stats.hints.total}`)
+      );
+    }
+    console.log('');
```

Comment:
Good catch on point #1.  I agree that we should print stats if there are _any_ messages.  I'll get that fixed.

On point #2, that was a conscious decision.  Since "info" and "hint" are "new" and right now you have to configure them to get any, I thought it best to keep the output uncluttered.  We could always change this later.